src/test/java/com/michelin/ns4kafka/integration/SchemaIntegrationTest.java

```diff
@@ -766,5 +766,53 @@ void shouldRegisterSameSchemaTwice() {
         assertNotNull(schemaAfterPostOnRegistry.id());
         assertEquals(2, schemaAfterPostOnRegistry.version());
         assertEquals("ns1-subject3-value", schemaAfterPostOnRegistry.subject());
+
+        // Delete latest schema version
```

Comment:
I think a dedicated integration test for deletion would be preferable:
- Register 4 schema versions
- Delete "latest" (v4). Check all schemas response
- Delete old version "v2". Check all schemas response
- Delete all (no version param). Check all schemas response
What do you think?

Reply:
I hesitated between having a dedicated tests or enriching the existing test. I just did the one that took less effort. I can write a dedicated one